tailwind.base.css

```diff
@@ -151,16 +151,20 @@
       font-medium text-[length:--text-size] text-[color:--text-color] bg-[color:--background-color] flex justify-center items-center content-center;
   }
   /* makes sure icons will not make the badges grow vertically; also make size proportional */
-  .gn-ui-badge mat-icon.mat-icon {
+  .gn-ui-badge ng-icon {
     margin-top: -0.325em;
     margin-bottom: -0.325em;
     flex-shrink: 0;
     vertical-align: middle;
-    font-size: 1.4em;
+    font-size: 1.1em;
     width: 1em;
     height: 1em;
   }
 
+  .gn-ui-badge ng-icon.badge-warning {
+    font-size: 1.4em;
+  }
```

Comment:
I had put it here since other classes for `gn-ui-badge` were here but if it's specific to the editor keywords then it makes more sense to move it :slightly_smiling_face:

Reply:
And it's required now because iconoir icons are overall smaller than mat-icons, so I had to increase their size :)

Reply:
that makes sense!!